CONTRIBUTING.md

```diff
@@ -0,0 +1,3 @@
+# Guide lines for contribuition in our gitbook!
+* [http://dev.docs.lightblue.io/development_process/README.html](http://dev.docs.lightblue.io/development_process/README.html)
+* Questions & suggestions,s end on nabble ->  http://forum.lightblue.io/ OR  http://dev.forum.lightblue.io/
```

Comment:
/s end/send/

And suggestions for contribution would be for the dev forum only.  Recommend making it a nice like like:

```
Questions & suggestions?  Share on our [dev forum](http://dev.forum.lightblue.io/)!
```